bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -50,6 +52,7 @@
 
 		<label>Compact Smart Plug</label>
 		<description>A compact smart plug with energy monitoring capabilities.</description>
+		<semantic-equipment-tag>PowerOutlet</semantic-equipment-tag>
```

Comment:
We might want to consider `Equipment_PowerOutlet_SmartPlug`, WDYT? English is not my primary language, but I think a power outlet is usually a fixed device installed in a wall to provide mains electricity? So a smart plug functionally acts as a controllable power outlet to the device plugged into it. But physically it’s not a fixed part of the home’s electrical infrastructure.

Reply:
From my point of view any kind of power outlet that can talk to OH is per definition 'smart'.

Reply:
My point is the opposite: Even if a power outlet is smart, that doesn't make it a smart plug. And a smart plug is not a power outlet, at least not physically.

Reply:
Just to clarify where I'm coming from. I have some of these installed in my house:

![image](https://github.com/user-attachments/assets/ce9aa6a1-0407-4db3-a4dc-93a3298ba409)

They are smart and in the wall. The traditionally/old one I replaced I would call "power outlet", and this one I would either call a "power outlet" as well, or a "smart power outlet" (here I agree that "smart" can be deduced, since it's in openHAB).

On the other hand, this external thing:

![image](https://github.com/user-attachments/assets/e0ab4455-f8d7-46f9-8c2c-1d5a2d84cae1)

I would call a "smart plug", not a "power outlet".

Reply:
So does it matter? (Similar diacussion about 'end' vs 'means' ..)

Reply:
Just to over exaggerate it: we could consider Power_Outlet_Mouneted_In_Wall_Having_Schuko_Pin_Layout_And_White_Color .. And_Smart ..